lib/DAC/DAC.h

```diff
@@ -12,7 +12,8 @@ enum pot_chan {
 };
 
 void init_dac(void);
-void set_pot(uint8_t val, pot_chan channel);
+//void set_pot(uint8_t val, pot_chan channel);
```

Comment:
Aren't we using the double potentiometer device any longer?

Reply:
reintegrate the DS1803 double potentiometer. It was switched because of experiments with single channel SPI poti